MBI rework (#4386)
centreon/centreon-documentation: versioned_docs/version-25.10/mbi-reworked/intro-to-mbi.md

---
id: mbi-intro
title: Introduction to MBI
---

> 

Author: à transformer en FAQ

## Overview

MBI is an extension/module that can be used to generate reports in many different formats. We have hundreds of report designs at your disposal that 
MBI is not originally included with Centreon and must be [installed separately(link) and requires a valid licence to use. Contact [Centreon support(link) to obtain a licence and start using Centreon.

> Note that MBI requires the user to respect specific settings and procedures to work, we highly recommend you read our documentation to avoid running into issues.

Author: specify that reports will not be generated if the user does not respect the procedures

MBI uses jobs to schedule the generation of reports using ou

## What does MBI do?

> brève description de ce que ça fait et lien vers topic "comment ça marche" (situé potentiellement dans troubleshooting)

Author: "comment ça marche" pourra potentiellement être son propre topic

## What are the possible outputs?
les potentiels résultats; lien vers les templates possibles formats de sorties (csv, pdf) et possibilité de le visualiser sur l'interface 

> 

Author: signaler que tous les rapports n'ont pas le même format de sortie, préparer une checklist

> brève description de ce que ça fait, comment ça marche, 

Author: ```suggestion
```

---
id: mbi-intro
title: Introduction to MBI
---

> The report generating capabilites of Centreon rely on the **Monitoring Business Intelligence (MBI)** extension.

Reviewer: please, please, do not use that sentence

> The report generating capabilites of Centreon rely on the **Monitoring Business Intelligence (MBI)** extension.

Author: ```suggestion
Centreon offers the possibility of creating regular reports on various metrics using the **Monitoring Business Intelligence (MBI)** extension.
```

> > Centreon BAM is a Centreon **extension** that requires a valid [license](../administration/licenses.md). To

Author: ```suggestion
> Centreon MBI is a Centreon **extension** that requires a valid [license](../administration/licenses.md). To
```
